src/Storages/IRemoteFileMetadata.cpp

```diff
@@ -0,0 +1,7 @@
+#include <Storages/IRemoteFileMetadata.h>
+namespace DB
+{
+
+IRemoteFileMetadata::~IRemoteFileMetadata() {}
```

Comment:
Looks strange to make `cpp` file just for this. Why not in `.h` file, `= default` should work fine.

Reply:
done